src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj

```diff
@@ -18,9 +18,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., build.props))\build.props" />
   <PropertyGroup>
     <!-- Override the TargetFrameworkVersion specified in build.props. This project must target
-         .NET 4.6 because the Sarif.Validator project, upon which this project depends, targets
+         .NET 4.5.2 because the Microsoft.Json.Schema package, upon which this project depends, targets
```

Comment:
can't JSON.Schema compile against 4.5.1? The rest of SARIF does. Also, we set this property in build.props, do you don't need TargetFrameworkVersion here at all (except to override it as you've done)